### PR TITLE
fix error in updating sharetools share-title

### PR DESCRIPTION
**Pull request by MelizzaP**
@daytonn @kand can you take a look at this?

### CODE REVIEW

File: elements/bulbs-video/elements/video-carousel.js [hunk 1]
```diff
@@ -120,7 +120,11 @@ class BulbsVideoCarousel extends BulbsHTMLElement {
 
     forEach.call(
       this.querySelectorAll('share-tools'),
-      (element) => element.setAttribute('share-url', this.state.shareUrl),
+      (element) => element.setAttribute('share-url', this.state.shareUrl)
```
Comment (kand) [inline elements/bulbs-video/elements/video-carousel.js]: I believe, according to the linting rules in this project, we need the trailing comma.

**Comment by kand** [comment]
Otherwise 👍

File: elements/bulbs-video/elements/video-carousel.js (hunk 1 above)
Comment (collin) [inline elements/bulbs-video/elements/video-carousel.js]: One annoyance is JS rejects a trailing comma in function invocation.

`console.log('hello', 'world',)` is a syntax error

**Comment by spra85** [comment]
Aside from other comments, 👍

**Pull request merged**